projects/storefrontlib/src/cms-components/checkout/components/promotions/promotions.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="cx-promotions" *ngIf="promotions">
-  <strong *ngFor="let promotion of promotions">
+  <ul *ngFor="let promotion of promotions">
```

Comment:
Changing the DOM structure is considered a breaking change. You can use feature lavels to introduce markup change.